src/main/java/com/dreammaster/scripts/ScriptChisel.java

```diff
@@ -224,6 +227,65 @@ public void loadRecipes() {
         ChiselHelper.addVariationFromStack("CobblestoneBricks", getModItem(Chisel.ID, "cobblestone", 1, 13, missing));
         ChiselHelper.addVariationFromStack("CobblestoneBricks", getModItem(Chisel.ID, "cobblestone", 1, 14, missing));
         ChiselHelper.addVariationFromStack("CobblestoneBricks", getModItem(Chisel.ID, "cobblestone", 1, 15, missing));
+        ChiselHelper.addGroup("diamond_block");
+        ChiselHelper.addVariationFromStack("diamond_block", getModItem(Minecraft.ID, "diamond_block", 1, 0, missing));
+        ChiselHelper.addVariationFromStack("diamond_block", getModItem(Chisel.ID, "diamond_block", 1, 1, missing));
```

Comment:
Why isn't block with meta 0 included into this group? And why don't you use `for` loop to simplify such repeated code?
Same for iron and gold.

Reply:
Some Chisel Blocks such as Iron, Diamond, and Gold start at meta 1 instead of 0. Not sure why that is

As for using for loops, I'm not sure why I didn't think of that. I'll throw that in

Reply:
Replaced with loops. I have refrained from touching the rest of the Scripts file which is unrelated to this PR